
# speed left right [m/sec], order is FRONT_LEFT, BACK_LEFT, FRONT_RIGHT, BACK_RIGHT
float32[4] speed



